src/connectedk8s/azext_connectedk8s/_params.py

```diff
@@ -56,3 +76,12 @@ def load_arguments(self, _):
         c.argument('cluster_name', options_list=['--name', '-n'], id_part='name', help='The name of the connected cluster.')
         c.argument('kube_config', options_list=['--kube-config'], help='Path to the kube config file.')
         c.argument('kube_context', options_list=['--kube-context'], help='Kubconfig context from current machine.')
+
+    with self.argument_context('connectedk8s proxy') as c:
+        c.argument('cluster_name', options_list=['--name', '-n'], id_part='name', help='The name of the connected cluster.')
+        c.argument('token', options_list=['--service-account-token'], help='Client authentication token for service account scenario.')
```

Comment:
>Client authentication token for service account scenario. [](start = 76, length = 57)

Let's just say - "Service account token to use to authenticate to the kubernetes cluster"

Reply:
I have updated the help statement.